modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -719,11 +732,66 @@ public void nodeWithDataTest() throws InterruptedException {
         checkTableWithData(ignite, TABLE_NAME);
     }
 
+    /**
+     * Restarts the node which stores some data.
+     */
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
```

Comment:
I think you should simply split this test in two

Reply:
Why? It's the same test basically

Reply:
ok, never mind